### ISSUE TITLE
Galaxy j6+ Android 10 32GB 3GB ??

### ISSUE THREAD

**Issue by moura464** [rafaelvcaetano/melonDS-android#185]
* **Device model:** SM-J610G
* **Android version:** Android API 29
* **melonDS version:** Beta 1.3.0

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$d: Failed to load firmware: BIOS9_MISSING
	at me.magnum.melonds.ui.emulator.h$c.a(Unknown Source:67)
	at f.a.b0.e.a.a.g(Unknown Source:10)
	at f.a.a.a(Unknown Source:14)
	at f.a.b0.e.a.e$a.run(Unknown Source:2)
	at f.a.r$a.run(Unknown Source:9)
	at f.a.b0.g.k.run(Unknown Source:13)
	at f.a.b0.g.k.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)

```

**Configuration:**  
```
FirmwareCrashContext(emulatorConfiguration=EmulatorConfiguration(useCustomBios=true, dsConfigDirectory=//, dsiConfigDirectory=//, internalDirectory=/data/user/0/me.magnum.melonds/files, fastForwardSpeedMultiplier=-1.0, useJit=false, consoleType=DS, soundEnabled=true, micSource=BLOW, firmwareConfiguration=FirmwareConfiguration(nickname=Gamesxxx, message=Hello!, language=1, favouriteColour=0, birthdayMonth=1, birthdayDay=1), rendererConfiguration=RendererConfiguration(videoFiltering=LINEAR, threadedRendering=true)), dsiBiosDirUri=content://com.android.providers.downloads.documents/tree/msd%3A468, dsBiosDirUri=content://com.android.providers.downloads.documents/tree/msd%3A473)
```

**Comment by rafaelvcaetano** [comment]
What happens if you put your BIOS files outside the Download directory? Does it work?

**Comment by moura464** [comment]
![Screenshot_20210315-191305_melonDS](https://user-images.githubusercontent.com/22099440/111228168-85cebc00-85c2-11eb-9404-039416533633.jpg)
Bios melods Android error 😔
Galaxy j6+ not

**Comment by shabbirmahmud** [comment]
Something is wrong with the bios directory.Try to put ur bios and roms files outside the download folder.Dont forget to update the rom and bios directory in emulator settings?If that dont work then turn off ur custom bios and try to load the rom again.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
